platform-includes/session-replay/install/javascript.nuxt.mdx

```diff
@@ -0,0 +1,3 @@
+The Replay integration is **already included** with the Nuxt SDK package.
```

Comment:
l: I'd be specific here just to avoid any potential confusion

```suggestion
The Replay integration is **already included** with the `@sentry/nuxt` SDK package.
```